lib/testrail.js

```diff
@@ -24,88 +23,107 @@ class TestRail {
 	async getCases(projectId, suiteId) {
 		try {
 			const res = await this.axios.get(`get_cases/${projectId}&suite_id=${suiteId}`);
+			output.log(`getCases: SUCCESS - the response data is ${JSON.stringify(res.data)}`);
 			return res.data;
 		} catch (error) {
 			const parsedError = error && error.response && error.response.data ? error.response.data.error : error;
-			output.error(`Cannot get cases for projectId:${projectId} & suiteId:${suiteId}, due to ${parsedError}`);
+			output.error(`getCases: ERROR - cannot get results for case ${caseId} on run ${runId} due to ${parsedError}`);
```

Comment:
maybe I'm wrong here, but is this the case that caseId or runId would be null here 😨

Reply:
ah no, unpushed, commit, sorry!